src/site/content/en/blog/quictransport/index.md

```diff
@@ -0,0 +1,324 @@
+
+```js
+const ws = await transport.createSendStream();
+const writer = ws.getWriter();
+writer.write(...);
+writer.write(...);
+await writer.abort();
+// Not all the data may have been written.
+```
+
+#### ReceiveStream
+
+A <code>[ReceiveStream](https://wicg.github.io/web-transport/#receivestream)</code> is initiated by the server. Obtaining a <code>ReceiveStream</code> is a two-step process for a web client. First, it calls <code>receiveStreams()</code>, which returns a <code>ReadableStream</code>. Each chunk of that <code>ReadableStream</code>, is, in turn, a <code>ReceiveStream</code> that can be used to read <code>Uint8Array</code> instances sent by the server.
+
+```js
+async function readFrom(receiveStream) {
+  const reader = receiveStream.readable.getReader();
+  while (true) {
+    const {done, value} = await reader.read();
+    if (done) {
+      break;
+    }
+    // value is a Uint8Array
+    console.log(value);
+  }
+}
+
+const rs = transport.receiveStreams();
+const reader = rs.getReader();
+while (true) {
+  const {done, value} = await reader.read();
+  if (done) {
+    break;
+  }
+  // value is an instance of ReceiveStream
+  await readFrom(value);
+}
+```
+
+You can detect stream closure using the <code>[closed](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStreamDefaultReader/closed)</code> promise of the <code>[ReadableStreamDefaultReader](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStreamDefaultReader)</code>. When the QUIC stream is [closed with the FIN bit](https://tools.ietf.org/html/draft-ietf-quic-transport-27#section-19.8), the <code>closed</code> promise is fulfilled after all the data is read. When the QUIC stream is closed abruptly (for example, by <code>[STREAM_RESET](https://tools.ietf.org/html/draft-ietf-quic-transport-27#section-19.4)</code>), then the <code>closed</code> promise rejects.
+
+```js
+// Assume an active receiveStream
+const reader = receiveStream.readable.getReader();
+reader.closed.then(() => {
+  console.log('The receiveStream closed gracefully.');
+}).catch(() => {
+  console.error('The receiveStream closed abruptly.');
+});
+```
+
+#### BidirectionalStream
+
+A <code>[BidirectionalStream](https://wicg.github.io/web-transport/#bidirectional-stream)</code> might be created either by the server or the client.
+
+Web clients can create one using `createBidirectionalStream()`, which returns a promise for a `BidirectionalStream`.
+
+```js
+const stream = await transport.createBidirectionalStream();
+// stream is a BidirectionalStream
+// stream.readable is a ReadableStream
+// stream.writable is a WritableStream
+```
+
+You can listen for a `BidirectionalStream` created by the server with `receiveBidirectionalStreams()`, which returns a `ReadableStream`. Each chunk of that `ReadableStream`, is, in turn, a `BidirectionalStream`.
```

Comment:
```suggestion
You can listen for a `BidirectionalStream` created by the server with `QuicTransportreceiveBidirectionalStreams()`, which returns a `ReadableStream`. Each chunk of that `ReadableStream`, is, in turn, a `BidirectionalStream`.
```

Reply:
I don't believe that change would be accurate. It's meant to be `receiveBidirectionalStreams()`.